config/locales/en/course/assessment/submission/logs.yml

```diff
@@ -0,0 +1,15 @@
+en:
+  course:
+    assessment:
+      submission:
+        logs:
+          index:
+            assessment_title: 'Assessment Title'
+            header: 'Access Logs'
+            ip_address: 'IP Address'
+            student_name: :'course.assessment.submission.submissions.submissions.student_name'
+            submission_session_id: 'Submission Session ID'
```

Comment:
@jeremyyap How about call the fields `submission token` and `user session token`, it's not really the ID of the session, but rather we generated.